certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/dns_rfc2136.py

```diff
@@ -89,26 +90,34 @@ def _cleanup(self, _domain: str, validation_name: str, validation: str) -> None:
     def _get_rfc2136_client(self) -> "_RFC2136Client":
         if not self.credentials:  # pragma: no cover
             raise errors.Error("Plugin has not been prepared.")
-        return _RFC2136Client(self.credentials.conf('server'),
-                              int(self.credentials.conf('port') or self.PORT),
-                              self.credentials.conf('name'),
-                              self.credentials.conf('secret'),
-                              self.ALGORITHMS.get(self.credentials.conf('algorithm'),
-                                                  dns.tsig.HMAC_MD5))
+
+        sign_query = False
+        if cast(str, self.credentials.conf('sign_query')).upper() == "TRUE":
+            sign_query = True
+
+        return _RFC2136Client(cast(str, self.credentials.conf('server')),
+                              int(cast(str, self.credentials.conf('port')) or self.PORT),
+                              cast(str, self.credentials.conf('name')),
+                              cast(str, self.credentials.conf('secret')),
+                              self.ALGORITHMS.get(self.credentials.conf('algorithm') or '',
+                                                  dns.tsig.HMAC_MD5),
+                              sign_query)
```

Comment:
This still crashes Certbot if `dns_rfc2136_sign_query` is absent from the credentials file. You can just replace `sign_query` with:

```python
(self.credentials.conf('sign_query') or '').upper() == "TRUE"
```

Reply:
I did test this with the statement removed from the credentials file and worked with no crashes on my end, but again, this seems to be a more elagant way of doing this. Thanks!